recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set version = "1.4.0rc1" %}
+{% set version = "1.4.0" %}
 {% set org = "cyclus" %}
-{% set branch = "develop" %}
+{% set branch = "master" %}
 {% set sha256 = "7a6bd1334f3fd13ddac73fbb7a5218b958b793b81bda26552e9c3abbf255a905" %}
```

Comment:
So this is actually a `sha256sum` of the tarball, like you'd do with md5.  You have to download the tarball and run the sha256sum command against it.  

@conda-forge/core are the people to notify about doc issues, in general.  Though in this case the docs are in conda-build http://conda.pydata.org/docs/building/meta-yaml.html#source-from-tarball